doc/tests/README.md

```diff
@@ -66,6 +66,23 @@ mod tests {
 }
 ```
 
+#### Frontend Unit Tests
```

Comment:
Please move other changes to another PR(s) to keep this focused on the decision.